test/demo/ios-62-63/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/build/payment-shim.js"></script>
+<script type="text/javascript">
+  var supportedInstruments = [{
```

Comment:
If we use `const` here, will it be converted?

Reply:
No. the demo files don't get compiled.